spec/filters/environment_spec.rb

```diff
@@ -21,23 +21,4 @@
       insist { subject["@metadata"]["newfield"] } == "hello world"
     end
   end
-
-  describe "does nothing on non-matching events" do
-    # The logstash config goes here.
-    # At this time, only filters are supported.
-    config <<-CONFIG
-      filter {
-        environment {
-          type => "foo"
```

Comment:
I know is not test related, but I wonder if  instead of removing the hole test, change it to show off the new syntax? does it makes sense?